pkg/app/pagewriter/pagewriter.go

```diff
@@ -3,15 +3,20 @@ package pagewriter
 import (
 	"fmt"
 	"net/http"
+
+	"github.com/microcosm-cc/bluemonday"
 )
 
+// HTML sanitization policy creation done once for lifespan of the program
+var sanitizer = bluemonday.StrictPolicy()
```

Comment:
Oh it's here I see! Is this standard practice to have this as a global?

Reply:
I think so - here was the explanation on the library: https://github.com/microcosm-cc/bluemonday#usage

> // Do this once for each unique policy, and use the policy for the life of the program
> // Policy creation/editing is not safe to use in multiple goroutines
> p := bluemonday.UGCPolicy()

Very much open to other HTML sanitizer libraries you prefer. I hadn't done it before in Golang and that was the most polished one that came up on Google 😅

Reply:
So something I punted on that maybe is worth doing to? There's a minor attack scenario where attackers attempt to use XSS in fields that are logged with the hope that an admin is using a bad logging system that is weak to XSS or something.

We have a few user-generated fields in our logging templates (User Agent, Request ID, Host I think) where we can potentially call this as well? But might be worth a separate PR.

Reply:
Hmm yes that's a good point, we should definitely be sanitizing anything we log, maybe leave that for a future PR as you've suggested though

Reply:
What do you think? Should we fully sanitize like I have here? Or just do the simple HTML escape? I guess we aren't trying to render user supplied markup (usual case for sanitization) -- so maybe escaping is better here?